cmd/hypper/testdata/empty/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
Im guessing this is a example chart to use for testing down the line no?

Reply:
Correct, could have been left out of the PR.